packages/lib/src/vis/tiles/api.ts

```diff
@@ -0,0 +1,50 @@
+import type { NdArray } from 'ndarray';
+import type { Vector2 } from 'three';
+
+import type { TextureSafeTypedArray } from '../heatmap/models';
+import type { Size } from '../models';
+
+export function getLevelsSize(
+  size: Size,
```

Comment:
Since we're dealing with two different sizes, I would maybe clarify that this is the `imageSize` (or `rootSize` or something if you end up doing it recursively).